src/main/java/com/acrolinx/sidebar/utils/MatchUtils.java

```diff
@@ -54,19 +55,21 @@ public static List<? extends AbstractMatch> filterDangerousMatches(
 
   public static List<? extends AbstractMatch> filterInsignificantWhitespaceMatches(
       List<? extends AbstractMatch> abstractMatches, String lastCheckedContent) {
+    final Pattern pattern = Pattern.compile("\\s{2,}", Pattern.DOTALL);
```

Comment:
you can even make this Pattern a private constant, no need to create a new instance every time this method is being called

Reply:
Extracted as constant